frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketItem/ticketItemThumbnail/ticketItemThumbnail.component.tsx

```diff
@@ -38,13 +37,12 @@ export const TicketItemThumbnail = ({ ticket }) => {
 		getTicketResourceUrl(teamspace, project, containerOrFederation, ticket._id, resourceId, isFederation) : null;
 
 	const goToViewpoint = (event) => {
-		if (!hasViewpoint) return;
 		event.stopPropagation();
-
 		TicketsCardActionsDispatchers.setSelectedTicket(ticket._id);
 		TicketsCardActionsDispatchers.setSelectedTicketPin(hasDefaultPin(ticket) ? ticket._id : null);
 		TicketsActionsDispatchers.fetchTicketGroups(teamspace, project, containerOrFederation, ticket._id);
```

Comment:
The code above is very similar to the code used in `onClickTicket` inside `ticketItem.component`. Maybe it can be extracted and passed to this component? Like this:

**ticketItem.component**:
```ts
	const selectTicket = (event) => {
		event.stopPropagation();
		TicketsCardActionsDispatchers.setSelectedTicket(ticket._id);
		TicketsCardActionsDispatchers.setSelectedTicketPin(hasDefaultPin(ticket) ? ticket._id : null);
		TicketsActionsDispatchers.fetchTicketGroups(teamspace, project, containerOrFederation, ticket._id, revision);
	};

	const onClickTicket = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
		TicketsCardActionsDispatchers.openTicket(ticket._id);
		selectTicket(event);
	};
```

in **ticketItemThumbnail.component**
```ts
export const TicketItemThumbnail = ({ ticket, onClick }) => {
	...

	const goToViewpoint = (event) => {
		onClick(event);
		
		if (!hasViewpoint) return;
		goToView(defaultView); 
	};
```